lib/features/composer/presentation/composer_controller.dart

```diff
@@ -1726,10 +1736,15 @@ class ComposerController extends BaseController
     if (bccRecipientState.value == PrefixRecipientState.disabled) {
       bccRecipientState.value = PrefixRecipientState.enabled;
     }
-    listBccEmailAddress = listEmailAddress.toList();
+    if (composerArguments.value?.emailActionType == EmailActionType.composeFromMailtoUri) {
+      listBccEmailAddress = {...listEmailAddress, ...?composerArguments.value?.bcc}.toList();
+    } else {
+      listBccEmailAddress = listEmailAddress.toList();
+    }
     toAddressExpandMode.value = ExpandMode.COLLAPSE;
     ccAddressExpandMode.value = ExpandMode.COLLAPSE;
     bccAddressExpandMode.value = ExpandMode.COLLAPSE;
+    bccAddressExpandMode.refresh();
```

Comment:
why do we need call refresh bcc?

Reply:
The UI won't update the number of bcc, as `bccAddressExpandMode` value does not change

Reply:
As I see the above command has assigned
```
    bccAddressExpandMode.value = ExpandMode.COLLAPSE;
```

Is this not enough to make Obx heard?

Reply:
Yes, because `bccAddressExpandMode.value` was `ExpandMode.COLLAPSE` before. Reassign it to the same value doesn't make the UI update.